test/tags.js

```diff
@@ -11,7 +11,11 @@ var liquid = Liquid({
     ctx, src, dst;
 
 function test(src, dst) {
-    expect(liquid.parseAndRender(src, ctx)).to.equal(dst);
+    liquid.parseAndRender(src, ctx)
```

Comment:
You should return this promise here, and finally return this promise in `.it()` function. Otherwise, Mocha will assume test success for undefined return value without Exception. See: <https://mochajs.org/#working-with-promises>